### PR TITLE
Fix loading of PICO_TOOLCHAIN_PATH

### LINKED ISSUE #258: Build option PICO_TOOLCHAIN_PATH has no effect
Was trying to use `PICO_TOOLCHAIN_PATH` to point to the location where I have it installed on my setup.
(I do _NOT_ have gcc it on my path)

Turns out that this option has no effect because it's always cleared [here](https://github.com/raspberrypi/pico-sdk/blob/26653ea81e340cacee55025d110c3e014a252a87/cmake/pico_pre_load_toolchain.cmake#L3).

As soon as I comment out this line, I can set the `PICO_TOOLCHAIN_PATH` option and `pico_find_compiler` happilly finds the compiler.

I can submit a PR with this fix if you want.

### PR DESCRIPTION
**Pull request by josesimoes**
- Add double quotes because build option it's a string.
- Remove comment as requested by @kilograham.
- Resolves #258.

### CODE REVIEW

**Comment by lurch** [comment]
Nice :+1:   Apologies if any of my uninformed comments threw you off the scent! :nose:

File: cmake/pico_pre_load_toolchain.cmake [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # PICO_CMAKE_CONFIG: PICO_TOOLCHAIN_PATH, Path to search for compiler, default=none (i.e. search system paths), group=build
-# Set your compiler path here if it's not in the PATH environment variable.
-set(PICO_TOOLCHAIN_PATH "" CACHE INTERNAL "")
+# Set your compiler path here with PICO_TOOLCHAIN_PATH CMake option, if it's not in the PATH environment variable.
```
Comment (kilograham) [inline cmake/pico_pre_load_toolchain.cmake:2]: I'd remove this comment completely - the PICO_CMAKE_CONFIG comment explains what the var is for.

**Review by kilograham — CHANGES_REQUESTED** [review]

File: cmake/pico_pre_load_toolchain.cmake (hunk 1 above)
Comment (josesimoes) [reply cmake/pico_pre_load_toolchain.cmake]: Done

**Comment by kilograham** [comment]
sorry, can you please rebase this onto develop-1.1.1 (it'll get released sooner that way) - still working thru the best branching policy!

**Comment by josesimoes** [comment]
@kilograham  I've started started over from develop1.1.1 and changed the base as develop1.1.1.
Trust this is how you wanted it...

**Review by kilograham — APPROVED** [review]

**Pull request merged**